chain/modules/svm/golana/sysvar.go

```diff
@@ -19,11 +27,122 @@ var DefaultRent = Rent{
 	BurnPercent:         types.DefaultBurnPercent,
 }
 
+type EpochSchedule struct {
```

Comment:
Let's remove all unused files from golana here, put all necessary struct types into types.go and only expose types.go